__tests__/app.test.js

```diff
@@ -29,6 +29,7 @@ describe("/api/article/:article_id", () => {
     test("200: Returns requested article", () => {
       const output = convertTimestampToDate({ ...data.articleData[2], article_id: 3 });
       output.created_at = output.created_at.toJSON();
+      output.comment_count = "2";
```

Comment:
when adding a new feature you should be creating a new test, not modifying the original test. If the original test is failing with the changes it's a sign that it's a brittle test.